packages/admin/src/components/stock/StockSearchSideBar.tsx

```
@@ -15,13 +40,13 @@ export const StockSearchSideBar = () => {
         <SearchInput inputText="종목 검색.." />
       </UpperWrapper>
       <ArticleWrapper>
-        <StockDiv />
-        <StockDiv />
-        <StockDiv />
-        <StockDiv />
-        <StockDiv />
+        {datas.map((data) => (
+          <StockDiv name={data.name} number={data.id} onClick={(id) => stockDivClick(data.id)}/>
+        ))
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)